pkgs/games/endgame-singularity/default.nix

```diff
@@ -0,0 +1,59 @@
+{ stdenv, fetchurl, unzip, python2Packages }:
+
+python2Packages.buildPythonApplication rec {
+  pname = "endgame-singularity";
+  version = "0.30c";
+  format = "other";
+
+  srcs =
+    [
+      (fetchurl {
+        url = "http://www.emhsoft.com/singularity/singularity-${version}-src.tar.gz";
+        sha256 = "13zjhf67gmla67nkfpxb01rxs8j9n4hs0s4n9lnnq4zgb709yxgl";
+      })
+      (fetchurl {
+        url = "http://www.emhsoft.com/singularity/endgame-singularity-music-007.zip";
+        sha256 = "0vf2qaf66jh56728pq1zbnw50yckjz6pf6c6qw6dl7vk60kkqnpb";
+      })
+    ];
+  sourceRoot = ".";
+
+  buildInputs = [ unzip ]; # The music is zipped
+  propagatedBuildInputs = with python2Packages; [ python pygame numpy ];
+
+  # Add shebang to main file
+  postPatch = ''
+    echo '#!${python2Packages.python}/bin/python'\
+         $'\n'$(cat singularity-${version}/singularity.py)\
+         > singularity-${version}/singularity.py
+  '';
+
+  # This is not an error: it needs both compilation rounds
+  buildPhase = ''
+    python -m compileall singularity-${version}
+    python -O -m compileall singularity-${version}
+  '';
+
+  installPhase = ''
+    install -Dm755 singularity-${version}/singularity.py  $out/share/singularity.py
+    install -Dm644 singularity-${version}/singularity.pyo $out/share/singularity.pyo
+    install -Dm644 singularity-${version}/singularity.pyc $out/share/singularity.pyc
+    cp -R singularity-${version}/code singularity-${version}/data $out/share/
+    cp -R endgame-singularity-music-007 $out/share/music
+  '';
+
+  # Tell it where to find python libraries
+  # Also cd to the same directory as the code, since it uses relative paths
+  postFixup = ''
+  makeWrapper "$out/share/singularity.py" "$out/bin/singularity" \
+        --set PYTHONPATH "$PYTHONPATH" \
+        --run "cd $out/share"
+  '';
+
+  meta = {
+    homepage = "http://www.emhsoft.com/singularity/";
+    description = "A simulation of a true AI. Go from computer to computer, pursued by the entire world. Keep hidden, and you might have a chance";
```

Comment:
This is a bit too long for `description`. Please split it into `description` and `longDescription`. It looks like the first sentence may be a good `description`, the rest may be `longDescription`. `description` should not have a dot at the end. Maybe use `A game about a true AI` to make clear it is a game.